js/calendar.js

```diff
@@ -439,16 +441,7 @@ if(!String.prototype.formatNum) {
 
 		data.events = this.getEventsBetween(start, end);
 
-		switch(this.options.view) {
```

Comment:
You mean I have to ignore it? If so I think you have to correct it and add new commits. I mean it have to be done from your side.